ui/summary.go

```diff
@@ -68,6 +68,10 @@ func VerifyTrendColumnStat(stat string) error {
 		return ErrStatEmptyString
 	}
 
+	if stat == "count" {
```

Comment:
hmm I dislike this approach, especially given the fact that we repeat it on line 90... I think `TrendColumns` should be refactored, so that instead of a single "default and possible (with the exception of `p()`) stats/columns", we have 2 different things: 
- "possible non-`p()` stats" (as a bonus, we could use a `map[string]func(s *stats.TrendSink) float64`, to avoid the stupid `for` loop below)
- default trend columns (a slice)

Reply:
@na-- Thanks, I agree this is not ideal.

An issue with using a `map` is the arbitrary order, and the current code depends a lot on indexes. But I'll see if there's a way of doing this cleanly without breaking anything :)

Reply:
The map would just be used as a `Set` of the possible non-`p()` trend columns, together with the lambdas or methods that generate and humanize the values. The actual columns and everything else would still be a slice

Reply:
@na-- Take a look at the top-most commit: e76e2a92.

I took some time to properly clean up `ui/summary.go`, with some additional changes to the ones you mentioned:
- avoiding usage of global state in favor of a `Summary` struct.
- splitting up the trend columns validation and column generation (previously `generatePercentileTrendColumn()`) into validation only (`Summary.Validate()`) and generation (`Summary.generateCustomTrendValueResolvers()`--yeah, naming suggestions welcome...). This is a cleaner design, but I wish we could do both in one pass.
- not exporting most symbols, since they're not used externally.

Let me know what you think, and if this is an improvement at all over the previous version. :)

Reply:
@na-- I think we can resolve this?